ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -41,6 +43,7 @@ object LedgerApiTestTool {
     println("Tests marked with * are run by default.\n")
     Tests.default.keySet.toSeq.sorted.map(_ + " *").foreach(println(_))
     Tests.optional.keySet.toSeq.sorted.foreach(println(_))
+    Tests.performanceTests.keySet.toSeq.sorted.foreach(println(_))
```

Comment:
I think this is confusing without a description of how you run them.

```suggestion
    println("Tests marked with * are run by default.\n")
    println("You can include extra tests with `--include=TEST-NAME`, or run all tests with `--all-tests`.")
    Tests.default.keySet.toSeq.sorted.foreach(println("  * " + _))
    Tests.optional.keySet.toSeq.sorted.foreach(println("    " + _))

    println("\nAlternatively, you can run performance tests.")
    println("Performance tests are not run by default, but can be run with `--perf-tests=TEST-NAME`.\n")
    println("Performance tests:")
    Tests.performanceTests.keySet.toSeq.sorted.foreach(println("    " + _))
```

I haven't tested if this compiles or makes any sense.